CA/storage/domain-pubkey/foo/pub_key.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN RSA PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoENjufLd92nSBZ866CGS
+zn7gG8vmjX7yVXwTzIOKlyXyoyfCro8NT2S9mtXakKJ+loyIHauogYyKrC2ANwT1
+lcAcssev8ksLvzi3kWONY2J/4eh5hZP3aFliqvDT6XY5MbEt5GCiSZ/DmouuSumr
+xA01xqRc7NfVnkMsoGLamD15A8eTqQUmcSeS/SPyhsbZXTUTGYZFDFuepMZFFMgW
+iB/J20hfpDMQc9Yzq1Bg9F5WTkV2jhFhMCUN9G+NGQTn5QANx8/oEATxgV60S2i5
+1xlCuDIWEdT2W5dTWJxVCE7suGI4h9OQFQSr2ORi2ASTaRT7zpa/mEe4YDuB+ubH
+TQIDAQAB
+-----END RSA PUBLIC KEY-----
```

Comment:
Can we not store `.pem` file in Github? Can you add documentation on how to generate it?

Reply:
Well this is domain public key stored in CA. Ca cannot generate them. I had the RSA key generation function before. But you deleted it as unused. If you wanna use it, you can refer to previous commit. As you said, it is not being used, so we shouldn't keep it there.

Reply:
I wrote a script to generate domain public key using openssl. Do you wanna take a look at that?

Reply:
That looks good! But we haven't implemented the communication between website daemon and CA. How about we leave the domain pub key here so we don't need to generate key and move to CA folder every time. Easier to do a demo or something. We can always delete it later

Reply:
Sounds good!